src/loaders/omeZarrReader.js

```diff
@@ -0,0 +1,51 @@
+import { openArray } from 'zarr';
+import ZarrLoader from './zarrLoader';
+
+async function getJson(path, key) {
+  const res = await fetch(`${path}${key}`);
+  if (res.status === 404) {
+    throw Error(`Key '${key}' does not exist on zarr store.`);
+  }
+  const json = await res.json();
+  return json;
+}
+
+// Mostly a javascript implementation of ome-zarr-py
+// https://github.com/ome/ome-zarr-py/blob/master/ome_zarr.py
+
+class OMEZarrReader {
```

Comment:
This should get documentation.

Reply:
It may even be worth starting a document, like for tiff, on how to generate data if that is possible.